daffodil-runtime1/src/main/scala/org/apache/daffodil/debugger/InteractiveDebugger.scala

```diff
@@ -1497,6 +1482,17 @@ class InteractiveDebugger(runner: InteractiveDebuggerRunner, eCompilers: Express
         }
       }
 
+      object InfoHidden extends DebugCommand with DebugCommandValidateZeroArgs {
+        val name = "hidden"
+        override lazy val short = "hid"
+        val desc = "display whether or not we're within a hidden state"
```

Comment:
"... within a hidden group." would better match DFDL spec terminology.

Reply:
Doesn't look like this was resolved.